src/components/textfield/textfield.ts

```diff
@@ -121,7 +145,7 @@ export class MTextfield extends ModulVue implements InputManagementData {
     }
 
     public get inputType(): MTextfieldType {
-        return !this.passwordAsText ? this.type : MTextfieldType.Text;
+        return !this.passwordAsText ? this.type === MTextfieldType.Number ? MTextfieldType.Telephone : this.type : MTextfieldType.Text;
```

Comment:
`
switch(this.type) {
    case MTextfieldType.Number: return MTextfieldType.Telephone;
    case MTextFieldType.PassWord: return this.passwordAsText ? MTextfieldType.Text : MTextfieldType.Password
    default: return this.type;
}
`

Reply:
Ouin, ça serait cool que ça soit plus lisible, parce que vite de même je ne comprends pas ce que ça fait.